packages/core/src/components/axes/axis.ts

```diff
@@ -188,6 +206,13 @@ export class Axis extends Component {
 			}
 
 			if (rotateTicks) {
+				if (numberOfTicksProvided === null) {
+					axis.ticks(this.getNumberOfFittingTicks(width, tickHeight, 3.5));
```

Comment:
What's `3.5`?

Reply:
It is the space ratio we want to keep between ticks, used to calculate how many ticks will fit the axis `size / (tickSize * spaceRatio)`.

Reply:
Seems like we should move it to `configuration.ts`

Reply:
Seems like we should move it to `configuration.ts`